objects/rct2ww/scenery_large/rct2.ww.3x3mantr.json

```diff
@@ -81,7 +81,8 @@
             "cs-CZ": "Mangrove",
             "ko-KR": "맹그로브 나무",
             "pl-PL": "Mangrowiec",
-            "ru-RU": "Дерево манго"
+            "ru-RU": "Дерево манго",
+            "pt-BR": "Árvore Selvagem 3"
```

Comment:
This doesn't look like a translation for "mangrove tree". Perhaps you took the Spanish translation as your source? It's important that you translate from en-GB and not from another language, as the translations in Wacky Worlds and Time Twister are often way off.

While we're at it, could you also change the nl-NL to ` Mangroveboom`?